docs/02-for-app-authors/02-requirements.md

```diff
@@ -30,6 +33,95 @@ Every repository must contain a manifest describing how to build the application
 
 This manifest may import other manifest files in the same repository. It's quite common to add the [shared-modules](shared-modules) repository as a submodule.
 
+#### Example manifest
+
+This assumes that upstream brings appdata, icons and desktop files and installs them via their post build install. In this case we're using `meson` as the buildsystem.
+
+<Tabs groupId="manifest-language" defaultValue="json" queryString>
+  <TabItem value="json" label="json" default>
+
+```json title="io.github.example.MyCoolApp.json"
+{
+  "id": "io.github.example.MyCoolApp",
+  // The runtime is the environment that the application will run in, so you can use dependencies it provides
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  // This is the command that will be run when the application is launched
+  "command": "mycoolapp",
+  // These are the permissions that the application needs
```

Comment:
Let's go into some more detail here, like in what situations you want to use what permission. It's not always immediately obvious.

Reply:
Not sure the example is the place for this, shouldn't that be in the flatpak docs?

Reply:
Then we should point there :) 

Having the example manifest be as verbose as reasonably possible is a good idea, since it would be one of the first things a new Flatpak packager would look for. We should make it a good example.

Reply:
Added